mediapipe/tasks/ios/test/vision/image_classifier/ImageClassifierTests.swift

```diff
@@ -0,0 +1,690 @@
+       image: mpImage,
+       timestampMs:0)
+    } catch {
+      assertEqualErrorDescriptions(
+        error,
+        expectedLocalizedDescription: """
+      The vision task is not initialized with live stream mode. Current \
+      Running Mode: Video
+      """)
+    }
+
+    do {
+      let imagClassifierResult = try imageClassifier.classify(
+        image: mpImage)
+      XCTAssertNil(imagClassifierResult)
+    } catch {
+      assertEqualErrorDescriptions(
+        error,
+        expectedLocalizedDescription: """
+      The vision task is not initialized with image mode. Current Running \
+      Mode: Video
+      """)
+    }
+  }
+
+  func testClassifyFailsWithCallingWrongApiLiveStreamInMode() throws {
+    let imageClassifierOptions =
+    try XCTUnwrap(
+      imageClassifierOptionsWithModelPath(
+        ImageClassifierTests.floatModelPath))
+    
+    imageClassifierOptions.runningMode = .liveStream
+    imageClassifierOptions.completion = {(
+      result: ImageClassifierResult?, 
+      error: Error?) -> () in
+    } 
+
+    let imageClassifier = try XCTUnwrap(ImageClassifier(options: 
+      imageClassifierOptions))
+
+    let mpImage = try XCTUnwrap(
+      imageWithFileInfo(ImageClassifierTests.multiObjectsImage))
+
+    do {
+      let imagClassifierResult = try imageClassifier.classify(
+       image: mpImage)
+      XCTAssertNil(imagClassifierResult)
+    } catch {
+      assertEqualErrorDescriptions(
+        error,
+        expectedLocalizedDescription: """
+      The vision task is not initialized with image mode. Current Running \
+      Mode: Live Stream
+      """)
+    }
+
+    do {
+      let imagClassifierResult = try imageClassifier.classify(
+        videoFrame: mpImage,
+        timestampMs: 0)
+      XCTAssertNil(imagClassifierResult)
+    } catch {
+      assertEqualErrorDescriptions(
+        error,
+        expectedLocalizedDescription: """
+      The vision task is not initialized with video mode. Current Running \
+      Mode: Live Stream
+      """)
+    }
+  }
+
+  func testClassifyWithVideoModeSucceeds() throws {
+    let imageClassifierOptions =
+    try XCTUnwrap(
+      imageClassifierOptionsWithModelPath(
+        ImageClassifierTests.floatModelPath))
+    
+    imageClassifierOptions.runningMode = .video
+
+    let maxResults = 3;
+    imageClassifierOptions.maxResults = maxResults
+  
+    let imageClassifier = try XCTUnwrap(ImageClassifier(options: 
+      imageClassifierOptions))
+
+    let mpImage = try XCTUnwrap(
+      imageWithFileInfo(ImageClassifierTests.burgerImage))
+    
+    for i in 0..<3 {
+      let imageClassifierResult = try XCTUnwrap(
+        imageClassifier.classify(
+          videoFrame: mpImage,
+          timestampMs: i))
+      try assertImageClassifierResult(
+        imageClassifierResult,
+        hasCategoryCount: maxResults,
+        andCategories: 
+          ImageClassifierTests.expectedResultsClassifyBurgerImageWithFloatModel
+      )
+    }
+  }
+}
```

Comment:
Can we also port `testClassifyWithQuantizedModelSucceeds`?

Reply:
Added